src/character.h

```diff
@@ -45,6 +45,7 @@
 #include "player_activity.h"
 #include "pldata.h"
 #include "point.h"
+#include "proficiency.h"
```

Comment:
You don't need to include `proficiency.h` to get the `proficiency_bonus_type` enum, you can forward declare it like
`enum class steed_type : int;` is below. (e.g. `enum class proficiency_bonus_type : int;`).

This header (character.h) is included everywhere, so including more stuff in it drags up the build time.